compiler/damlc/tests/daml-test-files/SetFieldParens.EXPECTED.renamed-daml

```diff
@@ -0,0 +1,14 @@
+File:    
+  SetFieldParens.daml
```

Comment:
Note that I tweaked the output of `daml damlc compile --ghc-option=-ddump-rn` to replace the absolute path that would go here with just the file name - otherwise the golden file wouldn't be consistent across systems.

Reply:
This didn't work since the windows absolute file name didn't trigger a newline like expected here. I'm now testing another approach where I just keep the part of the output under the `=== Renamer ===` title